src/DynamoCore/Services/StabilityTracking.cs

```diff
@@ -100,6 +118,7 @@ public static void WriteUptimeBeat(TimeSpan timespan)
         /// </summary>
         public static void Startup()
         {
+            StabilityUtils.IsLastShutdownClean = IsLastShutdownClean();
```

Comment:
Whether there is a crash in the last session is obtained from the registry. This value will be changed shortly after the cookie class's Startup method is called. This is why a property is used so that we can set the value and get the value later. A new static class is added because I do not want to mess up with the cookie class.